src/ServiceControlInstaller.PowerShell/Cmdlets/ServiceControlInstances/InstallServiceControlInstanceFromUnattendedFile.cs

```diff
@@ -1,89 +0,0 @@
-﻿// ReSharper disable UnassignedField.Global
-// ReSharper disable MemberCanBePrivate.Global
-// ReSharper disable UnusedAutoPropertyAccessor.Global
-
-namespace ServiceControlInstaller.PowerShell
-{
-    using System;
-    using System.IO;
-    using System.Management.Automation;
-    using Engine.Instances;
-    using Engine.Unattended;
-    using Engine.Validation;
-    using PathInfo = Engine.Validation.PathInfo;
-
-    [Cmdlet(VerbsCommon.New, "ServiceControlInstanceFromUnattendedFile")]
-    public class NewServiceControlInstanceFromUnattendedFile : PSCmdlet
```

Comment:
@mikeminutillo we looked back in history and saw that the deprecation warning was added in 2017 so we figured it is about time to remove this code. Do you see any problems with that? Anything we missed?

Reply:
No problem. Shoud that make this a major release technically?